composer.json

```diff
@@ -17,11 +17,16 @@
         "friendsofphp/php-cs-fixer": "^2.0",
         "phpunit/phpunit": "^9.5",
         "symfony/phpunit-bridge": "^3.0|^4.0"
+        "phpstan/phpstan": "^1.8"
     },
     "autoload": {
         "psr-4": { "Ivory\\OrderedForm\\": "src/" }
     },
     "autoload-dev": {
         "psr-4": { "Ivory\\Tests\\OrderedForm\\": "tests/" }
+    },
+    "scripts": {
+        "phpstan": "phpstan analyse --level=5 src",
+        "tests": "SYMFONY_DEPRECATIONS_HELPER=weak phpunit"
```

Comment:
I would appreciate an explanation for this, maybe in the commit message? If I understand it correctly, thanks to this setting, the deprecations are not shown in the output of the tests, just their count?

Reply:
you're right, that's the reason. I add an explanation to the commit message

Reply:
I have added the explanation to the first commit